docs/reference/announcements/880.md

```diff
@@ -123,25 +123,20 @@ Additional upgrade considerations are necessary for deployments that use custom
 
 ## Camunda Exporter <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
 
-A new Camunda Exporter brings the importer and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). The index schema is also being harmonized.
+Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporter.
 
-The exporter can consume Zeebe records (mostly events created by the engine), aggregate data, and store the related data into shared and harmonized indices.
+![current-state](../img/current-state-refined.png)
 
-- Data is archived in the background, coupled to the exporter but without blocking the exporter's progress.
-- Indices can be located in either Elasticsearch (ES) or OpenSearch (OS). Our web components (Tasklist and Operate) then use the new harmonized indices to show data to the user.
+In the past release, we’ve engineered a new Camunda Exporter, which brings the importer and archiving logic of web components (Tasklist and Operate) closer to our distributed platform (Zeebe). This allows us to simplify our installation, enable scalability for web apps, reduce the latency to show runtime and historical data, and reduce data duplication (resource consumption).
 
-The following diagram shows a simplified version of this work.
+![target](../img/target-brown-field-refined-without-optimize.png)
 
-![Camunda Exporter diagram](../img/target-camunda-exporter.png)
-
-- For example, Tasklist and Operate Importers are still required for old data to be imported, but the Camunda exporter writes all new data into ES/OS. After old indices are drained, importers can be turned off.
-- The archiver, which takes care of the archiving of completed process instances, will be moved into the Zeebe system as well, to reduce the installation complexity and provide a better scaling and replication factor (based on partitions).
-- This helps achieve a streamlined architecture, and improves platform performance and stability (especially regarding ES/OS).
-- A new separate component covers the migration, which will be part of the single application but can also deployed separately. It will adjust the previous Operate indices to make them more harmonized and usable by Tasklist.
+The new Camunda Exporter helps us achieve a more streamlined architecture, better performance, and stability (especially concerning ES/OS). The target release for the Camunda Exporter project is the 8.8 release.
+For more details about this project, you can also read our blog post [here](https://camunda.com/blog/2025/02/one-exporter-to-rule-them-all-exploring-camunda-exporter/).
 
 ### Harmonized index schema
 
-Camunda is harmonizing our index structure and usage.
+Our existing data schema in the secondary storage has been harmonized, to be used by all our components.
```

Comment:
We need to add documentation about `primary` and `secondary storage` in C8. @akeller would your team be able to help with that?

Reply:
Is there a specific epic tied to an alpha? @conceptualshark might be able to assist, but would need to partner with @ChrisKujawa or someone else involved with the project and have capacity. What kind of timeline are we looking at here?

Reply:
These docs won't be specifically related to the exporter topic, it's more of a general concept that we can refer to from exporter or rdbms topics. I think it would be good to pair with @romansmirnov to provide a clear explanation of these concepts. 
Would someone from your team be able to lead this effort?

Reply:
@conceptualshark do you have capacity for this? Delivered with 8.8, but might need a skeleton structure page in an upcoming alpha. Regardless, we'll need a docs issue for this since it won't be covered by either epic directly, but could be linked to both.

Reply:
@akeller I can definitely take this on - I made a [placeholder issue](https://github.com/camunda/documentation-team/issues/458), but would love some input on filling it out. I can link it to this issue, but are there other epics that it should relate to?